tests/Unit/Libraries/Cms/Plugin/CMSPluginTest.php

```diff
@@ -249,14 +251,16 @@ public function testNotLoadLanguageWhenExists()
      *
      * @return  void
      *
-     * @since   4.2.0
+     * @since   __DEPLOY_VERSION__
```

Comment:
```suggestion
     * @since   4.2.0
```
I don't think we should change the since tag here.